### ISSUE TITLE
Make place-holder usage consistent

### ISSUE THREAD

**Issue by elfring** [twaugh/system-config-printer#14]
I wonder a bit about the different usage of place-holders in [the provided udev rule file](https://github.com/twaugh/system-config-printer/blob/aa74679257e8822f3b855b62d1476933a016f828/udev/70-printers.rules#L1 "Update candidate").
* Is it important to express that only two characters may follow a known prefix?
* Does it matter if a specific filter pattern can be eventually checked faster by the involved software than the other specification (using an asterisk instead of question marks)?

**Comment by twaugh** [comment]
For consistency, we should probably use '??' for both cases, but I wouldn't expect it to make any difference in behaviour.

**Comment by elfring** [comment]
Would it be make sense to increase type safety eventually by restricting the affected filter expressions even to hexadecimal digits?

**Comment by twaugh** [comment]
Not really. In this context, we want it to be readable more than we mistrust `usb_id`.
